packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -49,12 +49,29 @@ const setDarkClass = (): void => {
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelectorAll('meta[name="theme-color"]')
+  const newThemeColor = window.getComputedStyle(
+    window?.document.documentElement
+  ).getPropertyValue(
+    isDark.value ? '--c-bg' : '--c-brand'
+  )?.trim()
+  themeColorEl?.forEach((mata) => {
+    mata.setAttribute('content', newThemeColor)
+  })
+}
+
+const setDarkMode = (): void => {
+  setDarkClass()
+  updateThemeColor()
+}
```

Comment:
Let's make the these methods name consistent. I think we can use `updateXXX`.

Reply:
https://github.com/vuepress/vuepress-next/blob/5399ebc21cb05f3495e2b9fb12b620ce40ee28dc/packages/%40vuepress/theme-default/src/client/components/ToggleDarkButton.vue#L69-L72

Ok, this seems would be fine.